packages/block-library/src/query/view.js

```diff
@@ -37,30 +29,11 @@ store( 'core/query', {
 			if ( isValidLink( ref ) && isValidEvent( event ) && ! isDisabled ) {
 				event.preventDefault();
 
-				// Don't announce the navigation immediately, wait 400 ms.
-				const timeout = setTimeout( () => {
-					ctx.message = ctx.loadingText;
-					ctx.animation = 'start';
-				}, 400 );
-
 				const { actions } = yield import(
 					'@wordpress/interactivity-router'
 				);
 				yield actions.navigate( ref.href );
 
-				// Dismiss loading message if it hasn't been added yet.
-				clearTimeout( timeout );
-
-				// Announce that the page has been loaded. If the message is the
-				// same, we use a no-break space similar to the @wordpress/a11y
-				// package: https://github.com/WordPress/gutenberg/blob/c395242b8e6ee20f8b06c199e4fc2920d7018af1/packages/a11y/src/filter-message.js#L20-L26
-				ctx.message =
-					ctx.loadedText +
-					( ctx.message === ctx.loadedText ? '\u00A0' : '' );
-
-				ctx.animation = 'finish';
-				ctx.url = ref.href;
```

Comment:
Is this line meant to be deleted?

Reply:
Nope. Fixed in https://github.com/WordPress/gutenberg/pull/58377/commits/56225f5c9f5c4a3658533d86279d257121e37635.